src/repos/UserRepo.ts

```diff
@@ -85,7 +86,10 @@ const updateUser = async (
 };
 
 const getUserByNetID = async (netID: string): Promise<User> => {
-  const user = await db().findOne({ netID });
+  const user = await db().findOne({
+    where: { netID: netID }, relations: ["matchings", "matchings.users",
+      "matchings.schedule", "matchings.schedule.times"]
+  });
   if (user == null) {
```

Comment:
ik this isn't your change but might wanna watch out with checking none types in javascript